openwisp_network_topology/api/views.py

```diff
@@ -39,46 +47,54 @@ class RequireAuthentication(APIView):
         ]
 
 
-class NetworkCollectionView(generics.ListAPIView, RequireAuthentication):
+class ListViewPagination(pagination.PageNumberPagination):
+    page_size = 10
+    page_size_query_param = 'page_size'
+    max_page_size = 100
```

Comment:
Users may want to configure this.
Also, do we need to set`page_size_query_param`, isn't `page_size` the default? (just asking, it's okay here!)

Reply:
@atb00ker when we override this class, then this parameter is not preent, we have at declare it, I tried undoing changes but didn't found it by.